templates/checks.json

```diff
@@ -0,0 +1,11 @@
+{
+  "checks": { {% for check in checks %}
+    "{{check.type}}": {
+      "command": "{{check.script}}",
+      "standalone": true,
+      "interval": 5,
+      "type": "metric",
+      "subscribers": ["{{check.subscribers}}"]
+    }{%- if not loop.last %},{% endif %}{% endfor %}
```

Comment:
Having this on the same line is error-prone because now `{%` is right next to `}` which makes it hard to see if the bracket is part of the json or of the template, easy to make mistakes when you edit this template: 

```
}{%- if not loop.last %},{% endif %}{% endfor %}
```

Also, instead of messing with templates, couldn't you just create a python dict and serialize that to json?